### PR TITLE
Add database config examples for all supported databases

### LINKED ISSUE #1676: Database config examples for all supported databases
https://github.com/openSUSE/osem/blob/master/config/database.yml.example only shows the config for sqlite3.

Since we support mysql and postresql, I would add comments with the configuration for each of those databases as well.

### PR DESCRIPTION
**Pull request by rishabh92**
Example for MySQL and PostgreSQL added.
Fixes #1676

### CODE REVIEW

**Comment by differentreality** [comment]
Nice :-) 
Please comment out the new lines though so that default functionality doesn't change (sqlite is used by default) and please move the code higher after the development block for sqlite.

**Comment by rishabh92** [comment]
@differentreality Please review the changes.

File: config/database.yml.example [hunk 1]
```diff
@@ -9,6 +9,35 @@ development:
   pool: 5
   timeout: 5000
 
+## PostgreSQL
+##   gem install pg
+## 
+##   Ensure the postgres gem is defined in your Gemfile
+##   gem 'pg'
+##   Update your credentials for username and password
```
Comment (differentreality) [inline config/database.yml.example]: If we are saying this, it needs to also mention database_name

File: config/database.yml.example [hunk 2]
```diff
@@ -9,6 +9,35 @@ development:
...
+##   gem 'mysql2'
+##   Update your credentials for username and password
+# development:
+#   adapter: mysql2
+#   encoding: utf8
+#   database: database_name
+#   pool: 5
+#   username: username
+#   password: password
+#  socket: /tmp/mysql.sock
```
Comment (differentreality) [inline config/database.yml.example]: One more space needed here

**Comment by differentreality** [comment]
Any particular reason why you removed timeout option for mysql and postgresql?

Otherwise this looks good, please make the style changes I mentioned so that we can merge this!

Normally you should push new changes to an open PR into a new commit, but feel free to leave 1 commit, and make sure you have rebased. Thank you :)

**Comment by rishabh92** [comment]
@differentreality timeout option is not mentioned in the [Ruby on Rails guide](http://guides.rubyonrails.org/configuring.html#configuring-active-record).
I have made the changes. Thanks :)

**Comment by differentreality** [comment]
@rishabh92 as mentioned in my previous comment, please keep just 1 commit.
Do you know how you can squash your 2 commits into 1?

**Comment by rishabh92** [comment]
@differentreality i think i misread your last comment. I have squashed my commit now. :)

**Comment by differentreality** [comment]
Nice @rishabh92 :+1:  thank you!

**Pull request merged**